flink-connector-elasticsearch8/src/main/java/org/apache/flink/connector/elasticsearch/sink/Elasticsearch8AsyncWriter.java

```diff
@@ -102,13 +103,15 @@ public Elasticsearch8AsyncWriter(
         final SinkWriterMetricGroup metricGroup = context.metricGroup();
         checkNotNull(metricGroup);
 
+        this.numRecordsSendCounter = metricGroup.getNumRecordsSendCounter();
         this.numRecordsOutErrorsCounter = metricGroup.getNumRecordsOutErrorsCounter();
     }
 
     @Override
     protected void submitRequestEntries(
             List<Operation> requestEntries, Consumer<List<Operation>> requestResult) {
-        LOG.debug("submitRequestEntries with {} items", requestEntries.size());
+        LOG.info("submitRequestEntries with {} items", requestEntries.size());
```

Comment:
debug -> info

Reply:
Understood. Though this gets called for a batch of entries, it might get many of this successful log message that are less useful. We can get this back to debug level; so it's more like "no-logs-is-good-logs". Or do you think it's a valid idea to print successful message at a frequency, so it's still explicitly reporting progress?

Reply:
I added a new commit to illustrate the latter idea. I can switch to DEBUG if you think it's overkill or not necessary.

Reply:
Sounds good. I have updated the code, but keeping the metric which I find useful and less intrusive.